### ISSUE TITLE
Fix link to python-novice-inflammation-data.zip

### ISSUE THREAD

**Issue by somexpert** [swcarpentry/python-novice-inflammation#297]
I was just looking over your lessons and noticed that the [setup instructions](https://github.com/swcarpentry/python-novice-inflammation/blob/gh-pages/setup.md) contain a broken link for downloading the required data in a zip file. I believe that it might need to be changed the same way [this was](https://github.com/swcarpentry/shell-novice/commit/438f900a13ec60c364845b76c45265dbcd093f19) in order to get the link to point to the correct zip file in the `/data` directory.

So, to summarize, I believe that:
`[python-novice-inflammation-data.zip](./python-novice-inflammation-data.zip)`
should be changed to:
`[python-novice-inflammation-data.zip]({{ site.github.url }}/data/python-novice-inflammation-data.zip)`
in setup.md.
